os/arch/arm/src/armv7-r/arm_assert.c

```diff
@@ -920,7 +921,7 @@ static void _up_assert(int errorcode)
 {
 	/* Are we in an interrupt handler or the idle task? */
 
-	if (g_upassert || current_regs || (this_task())->pid == 0) {
+	if (current_regs || (this_task())->pid == 0) {
```

Comment:
Add Macro dependency for fault manager case,

Reply:
sure, I will modify accordingly